src/controllers/client.controller.ts

```diff
@@ -18,9 +19,14 @@ export default class ClientController {
   public async createClient(req: Request, res: Response) {
     try {
       const clientData = req.body as clientInput;
-      const response = await this.repository.createClient(clientData);
+      const clientDataWithHash = { ...clientData, password_hash: 'whatever' }; // WIP: hashing
 
-      return res.cookie('jwt', response.token).status(201).json(response);
+      const newClient = await this.repository.createClient(clientDataWithHash);
+
+      const clientToken = JWTManager.generateToken(newClient);
```

Comment:
You are breaking single responsability here, abstract the token generation in another function

Reply:
In my next commit, tokens are no longer created in the user creation endpoint. thx for the review.